packages/client/src/updater.ts

```diff
@@ -404,6 +404,7 @@ export class Updater {
     return path.join(this.targetDir, filePath);
   }
 
+  /* eslint-disable @typescript-eslint/require-await */
   private async persistMetadata(metaDataName: string, bytesData: Buffer) {
```

Comment:
I think this may be another case where the linter is pointing out a mistake. I don't think this function needs to be `async` at all.

Reply:
true. this func shouldn't be async. i guess this linter is kind of catching this mistake too